packages/react-components/react-tree/stories/Tree/OpenItemControlled.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import {
+  Tree,
+  TreeItem,
+  TreeItemLayout,
+  TreeItemOpenChangeData,
+  TreeItemOpenChangeEvent,
+} from '@fluentui/react-components';
+
+export const OpenItemControlled = () => {
+  const [open, setOpen] = React.useState(true);
+  const handleOpenChange = (event: TreeItemOpenChangeEvent, data: TreeItemOpenChangeData) => {
+    setOpen(data.open);
+  };
+  return (
+    <Tree aria-label="Open Item Controlled">
+      <TreeItem open={open} onOpenChange={handleOpenChange} itemType="branch" value="tree-item-1">
```

Comment:
what happens when a user uses the both the Tree and TreeItem controlling at the same time?

Reply:
https://github.com/microsoft/fluentui/pull/29081/files#diff-05364b7856598e7414bb2dbde291faadcb7a9af0f90575a47c10d64afca1738bR48-R56

As stated in comment, tree item open state is an override, so it takes precedence.